ansible/roles/atmo-realvncserver/tasks/main.yml

```diff
@@ -98,15 +99,17 @@
         dest: /usr/share/backgrounds/default.jpg
         remote_src: True
       when: ansible_distribution == "Ubuntu" and ansible_distribution_major_version < "16"
+      ignore_errors: yes
 
     - name: Change desktop background on Ubuntu 16+
       copy:
         src: /usr/share/backgrounds/cloud_background.jpg
         dest: /usr/share/backgrounds/xfce/xfce-teal.jpg
         remote_src: True
       when: ansible_distribution == "Ubuntu" and ansible_distribution_major_version >= "16"
+      ignore_errors: yes
```

Comment:
My understanding: `failed_when: false` hides any failure, so the task returns OK/changed rather than failed in all cases. `ignore_errors: yes` will show the correct "failed" return status when a task fails, but it will not halt execution of subsequent tasks in the role/playbook.

Here, we may want to see when the task fails, but we still want the instance deployment to proceed to completion.